dips/dip-10.md

```diff
@@ -101,130 +92,62 @@ struct DiemIdDomainEvent {
   * `domain` - exact copy of `DiemIdDomain` that was added/removed from `DiemIdDomains` resource of a VASP account
   * `address` - address of an account where `DiemIdDomain` was added or removed
 
-# Off-chain protocol
-
-**TODO: This section may change in the future as we update DIP 1**
-
-<img alt="Payment flow" src={useBaseUrl('img/diem-id-flow.png')} />
-
-In order to support peer-2-peer payments, VASPs need to implement two Diem ID endpoints:
 
-* **Info endpoint** for querying optional information about the receiver. VASPs can provide this endpoint to improve UX and security of payment, but they are not obligated to - VASPs may choose not to share any information about the user via this endpoint, including whether the user exists or not.
-  * *For example*: VASP can define a 'contact list' and only share information if the sender is in receiver's contacts
-* **Payment endpoint** to perform actual money transfer.
+# ReferenceID Exchange
+The PaymentMetadata transactions require a ReferenceID in order to submit a transaction to the chain. PaymentMetadata transactions reveal nothing distinctly about either the sender or recipient and do not create a linkability between the sender or receiver across payments. Hence, the ReferenceID must be established off-chain and this protocol defines one such way to do so leveraging the PaymentAddresses.
 
-## Info endpoint
+## ReferenceID Command
+A VASP intending to send a payment from one VASP to another when leveraging Diem IDs constructs a ReferenceIDCommand submits that to the recipient side. If the recipient knows the recipient PaymentAddress and can potentially accept payments, it returns a success. In the case it cannot, it returns an error indicating the reason.
```

Comment:
fix naming I see both PaymentAddress and Diem ID, plus is it PayAddress?

Reply:
Pay Address, i just pushed a PR to fix all of it